mass_merge/models/__init__.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
```

Comment:
In Odoo 14.0 (since 11.0) we should no longer have coding lines in python files.